standard.go

```diff
@@ -461,7 +426,14 @@ func (r StandardRuleset) Settings() Settings {
 	}
 }
 
-// Adaptor for integrating stages into StandardRuleset
-func (r *StandardRuleset) callStageFunc(stage StageFunc, boardState *BoardState, moves []SnakeMove) (bool, error) {
-	return stage(boardState, r.Settings(), moves)
+// impl Pipeline
+func (r StandardRuleset) Error() error {
+	return nil
+}
+
+// IsInitialisation checks whether the current state means the game is initialising.
+func IsInitialisation(b *BoardState, settings Settings, moves []SnakeMove) bool {
```

Comment:
Slightly ridiculous nitpick, sorry - the rest of the codebase mostly uses `initialization`/`initialize` and we generally standardize on the American style spelling for public stuff.

Reply:
Good to know, I'll make sure to keep that in mind from now on - thank you!